### DISCUSSION TITLE
Firebase Query Error: function () { const temp = Object.assign({}, this); if (temp.credentials)

### DISCUSSION THREAD

**Discussion by 1stthingapps** [Budibase/budibase#14080] [Help]
Hi All- brandnewbie here. 

Trying out Budibase and trying to connect to our existing Firebase DB. I've managed to connect to collections in a brand new project in firebase but when I try to connect to the existing project I get the error:

**"Query Error: function () { const temp = Object.assign({}, this); if (temp.credentials)"** 

flashing up on screen.

I'd assume that this is a firebase issue- maybe something to do with authentication- but the error message seems to be a TypeScript or JavaScript message and that I assume is from Budibase.

Any ideas?

**Comment by niko2020** [comment]
same here, have you found a solution?
